### ISSUE TITLE
Microsoft tunnel gateway server health status fails due to matrix 'Internal network accessibility's is shown unhealthy

### ISSUE THREAD

**Issue by Abhishek-EH** [MicrosoftDocs/memdocs#2516]
The endpoint service being configured in a tunnel site is reachable over the internet, however in tunnel server health status tab, matrix 'internal network accessibility' is showing with status 'unhealthy', which causing overall tunnel server node status is being displayed
<img width="756" alt="Screenshot 2022-03-05 at 9 36 13 PM" src="https://user-images.githubusercontent.com/6381671/156891161-f7c8f86c-9550-463a-ae04-2c9cda3799dc.png">
 to be 'unhealthy'.
<img width="781" alt="Screenshot 2022-03-05 at 9 34 29 PM" src="https://user-images.githubusercontent.com/6381671/156891098-e8f6436f-a68d-4c9f-b33b-996839432108.png">
<img width="593" alt="Screenshot 2022-03-05 at 9 35 07 PM" src="https://user-images.githubusercontent.com/6381671/156891122-32cf5b3d-862f-44fb-8c40-ff5fecc93773.png">

**Comment by officedocspr5** [comment]
To make it easier for you to submit feedback on articles on learn.microsoft.com, we're transitioning our feedback system from GitHub Issues to a new experience. 

As part of the transition, this GitHub Issue will be moved to a private repository. We're moving Issues to another repository so we can continue working on Issues that were open at the time of the transition. When this Issue is moved, you'll no longer be able to access it.

If you want to provide additional information before this Issue is moved, please update this Issue before December 15th, 2023.

With the new experience, you no longer need to sign in to GitHub to enter and submit your feedback. Instead, you can choose directly on each article's page whether the article was helpful. Then you can then choose one or more reasons for your feedback and optionally provide additional context before you select **Submit**.

Here's what the new experience looks like.

**Note**: The new experience is being rolled out across learn.microsoft.com in phases. If you don't see the new experience on an article, please check back later.

First, select whether the article was helpful:

![Image showing a dialog asking if the article was helpful with yes and no answers.](https://learn.microsoft.com/Office/media/internal/new-feedback-yes-no.jpg)

Then, choose at least one reason for your feedback and optionally provide additional details about your feedback:

| Article was helpful | Article was unhelpful |
|:--|:---|
|![Image showing a dialog asking how the article was helpful with several options.](https://learn.microsoft.com/Office/media/internal/new-feedback-helpful.jpg) | ![Image showing a dialog asking how the article wasn't helpful with several options.](https://learn.microsoft.com/Office/media/internal/new-feedback-unhelpful.jpg)|

Finally, select **Submit** and you're done!
